### PR TITLE
feat(experimental): add new package

### PR DESCRIPTION
**Pull request by waterplea**
## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Refactoring
- [ ] Build or CI related changes
- [ ] Tests related changes
- [ ] Documentation content changes

## What is the current behavior?

Closes #

### CODE REVIEW

File: projects/experimental/README.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Taiga UI — Layout
```
Comment (vladimirpotekhin) [inline projects/experimental/README.md:1]: experimental

Comment (vladimirpotekhin) [reply projects/experimental/README.md:1]: (replace in whole file)

File: projects/demo/src/modules/experimental/avatar/examples/import/insert-template.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+```html
+<tui-avatar text="Batman"></tui-avatar>
```
Comment (vladimirpotekhin) [inline projects/demo/src/modules/experimental/avatar/examples/import/insert-template.md:2]: src or ng-content

File: projects/demo/src/modules/experimental/avatar/avatar.component.ts [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+    readonly example2: TuiDocExample = {
+        TypeScript: import('./examples/2/index.ts?raw'),
+        HTML: import('./examples/2/index.html?raw'),
+        LESS: import('./examples/2/index.less?raw'),
+    };
+
+    readonly example3: TuiDocExample = {
+        TypeScript: import('./examples/2/index.ts?raw'),
+        HTML: import('./examples/2/index.html?raw'),
```
Comment (vladimirpotekhin) [inline projects/demo/src/modules/experimental/avatar/avatar.component.ts:33]: examples/3/index

File: projects/experimental/components/avatar-stack/avatar-stack.module.ts [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+} from '@taiga-ui/experimental/components/avatar';
+
+import {TuiAvatarStackComponent} from './avatar-stack.component';
+
+@NgModule({
+    imports: [TuiAvatarModule],
+    declarations: [TuiAvatarStackComponent],
+    exports: [TuiAvatarStackComponent, TuiAvatarComponent],
+})
+export class TuiAvatarStackModule {}
```
Comment (nsbarsukov) [inline projects/experimental/components/avatar-stack/avatar-stack.module.ts:14]: It is not obvious from the documentation that we should import this new module.

Should we add description to [this example](https://taiga-ui--pr5090-experimental-32tiixoj.web.app/experimental/avatar#stacking)?

Why `TuiAvatarStackComponent` cannot be part of `TuiAvatarModule` ?

File: projects/experimental/components/avatar-stack/avatar-stack.style.less [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+            calc(var(--radius) + var(--gap)) calc(var(--radius) + var(--gap)),
+            calc(var(--radius) + var(--gap)) calc(var(--radius) + var(--gap)),
+            100%;
+        mask-repeat: no-repeat;
+    }
+
+    &[data-direction='left'] tui-avatar:not(:first-child) {
+        mask-image:
+            radial-gradient(circle at 0% 100%, transparent calc(var(--radius) + var(--gap)), #000 calc(var(--radius) + var(--gap) + 0.2px)),
+            radial-gradient(circle at 0% 0%, transparent calc(var(--radius) + var(--gap)), #000 calc(var(--radius) + var(--gap) + 0.2px)),
```
Comment (nsbarsukov) [inline projects/experimental/components/avatar-stack/avatar-stack.style.less:19]: lgtm))))

**Review by nsbarsukov — APPROVED** [review]

**Pull request merged**